### ISSUE TITLE
[18.10] "Display all Services for this host" doesn't filter

### ISSUE THREAD

**Issue by cgagnaire** [centreon/centreon#6926]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 18.10

**Centreon Engine version**: 18.10

**Centreon Broker version**: 18.10

**OS**: CentOS 7

**Additional environment details (AWS, VirtualBox, physical, etc.):**
VMware

**Steps to reproduce the issue:**
1. From 'Configuration > Hosts' page, click on the "Display all Services for this host" symbol

**Describe the results you received:**
Once redirected to 'Configuration > Services' page, there is no filter on the host even though the URL has the hostname for "search_h" parameter.

**Describe the results you expected:**
The "Hosts" field should be filled up with the hostname.

**Additional information you think important (e.g. issue happens only occasionally):**

**Comment by lpinsivy** [comment]
using 18.10.2 I can't reproduce this issue, see print screen ![Sans titre](https://user-images.githubusercontent.com/3351916/54218692-55914600-44ee-11e9-8eab-e35c5892ce8c.png)

**Comment by lpinsivy** [comment]
Workaround waiting new release:

```
--- /usr/share/centreon/www/include/configuration/configObject/host/listHost.ihtml.origin	2019-03-12 17:58:49.569000000 +0100
+++ /usr/share/centreon/www/include/configuration/configObject/host/listHost.ihtml	2019-03-12 17:59:05.958000000 +0100
@@ -67,7 +67,7 @@
 			<a href="{$elemArr[elem].RowMenu_link}"><img src="{$elemArr[elem].RowMenu_icone}" class="ico-18  margin_right" />{$elemArr[elem].RowMenu_name}</a>
 		</td>
 		<td class="ListColCenter">
-            <a href='./main.php?p=602&search_h={$elemArr[elem].RowMenu_name}&search_s='>
+            <a href='./main.php?p=602&searchH={$elemArr[elem].RowMenu_name}'>
 		        <img src="./img/icons/all_services.png" class="ico-16" title='{$HelpServices}'>
 		    </a>
 		</td>
```

**Comment by adr-mo** [comment]
Hi @cgagnaire, @lolokai 

Thanks for the feedback 💯 

This issue will be fixed in the **18.10.5 release** thanks to this Pull Request https://github.com/centreon/centreon/pull/7388

Regards,
